docs/tour/hello-world.mdx

```diff
@@ -25,14 +26,54 @@ If you have not installed `wash`, follow the [installation guide](../installatio
 
 :::
 
-In this guide, we'll be taking a tour through some of the most common activities in the wasmCloud
-ecosystem like starting and configuring the two main components of wasmCloud: [actors](/docs/reference/glossary#actors) and
-[capability providers](/docs/reference/glossary#capabilities). All wasmCloud applications need a [host](/docs/category/host-runtimes) to run on, so let's
+In this guide, we'll be taking a tour through the tooling in the wasmCloud ecosystem and a "Hello world" example. Once you're finished, you'll have a good understanding of how to use `wash` and `wadm` to build and deploy wasmCloud applications. All wasmCloud applications need a [host](/docs/concepts/hosts) to run on, so let's
 use `wash` to start one now.
 
-## Starting a Host
+## Starting Host Infrastructure
 
-To start a host, simply run `wash up`. The default settings should work well for this tutorial. The host can be killed at any time with `CTRL+c`
+<Tabs groupId="env" queryString>
+  <TabItem value="local" label="Local">
+
+To start a host, simply run `wash up`. The default settings should work well for this tutorial. The host can be killed at any time with `CTRL+c`.
+
+  </TabItem>
+  <TabItem value="docker" label="Docker">
+
+  You can use the following `docker-compose.yml` file to start the necessary host infrastructure in Docker:
```

Comment:
Should we link to https://github.com/wasmCloud/wasmCloud/blob/main/examples/docker/docker-compose-full.yml so we don't have to keep these in sync?